pkgs/applications/graphics/tesseract/tesseract5.nix

```diff
@@ -1,5 +1,7 @@
 { lib, stdenv, fetchFromGitHub, autoreconfHook, autoconf-archive, pkg-config
-, leptonica, libpng, libtiff, icu, pango, opencl-headers, fetchpatch }:
+, leptonica, libpng, libtiff, icu, pango, opencl-headers, fetchpatch
+, darwin
```

Comment:
This is not how we normally do things, however in this case it's a solution that doesn't require complicated rewrite of https://github.com/NixOS/nixpkgs/blob/07d567ed0ae7981fab5b0dae27da9ebca9fd7ed5/pkgs/applications/graphics/tesseract/wrapper.nix

Passing the frameworks separately the way we do it in all-packages requires either:
- the wrapper to ignore them for tesseract3/4,
- tesseract3/4 to accept them and leave unused, or
- using `...` to ignore them

Neither of those seem better than making an exception for accepting darwin here.

Feel free to suggest other solutions though.

Reply:
> it's a solution that doesn't require complicated rewrite



> using `...` to ignore them

Adding two ... is not complicated.

Reply:
Turns out I made it way harder than it needed to be. No `...`s necessary.